### PR TITLE
update geoip config

### PR DESCRIPTION
**Pull request by YamatoSecurity**
Updated geoip config. also slightly updated channel eid info

### CODE REVIEW

File: config/geoip_field_mapping.yaml [hunk 1]
```diff
@@ -3,6 +3,31 @@ SrcIP:
...
+    - 4648 # Explicit Logon %IpAddress%
+    - 4768 # Kerberos TGT Requested %IpAddress%
+    - 4769 # Kerberos Service Ticket Requested %IpAddress%
+    - 4778 # RDP Session Reconnect %ClientAddress%
+    - 4779 # RDP Session Disconnect %ClientAddress%
+    - 4825 # RDP Logon Failed %ClientAddress%
+    - 5140 # A network share object was accessed %IpAddress%
+    - 5145 # NetShare File Access %IpAddress%
+    - 5156 # The Windows Filtering Platform has allowed a connection %SourceAddress%
+  - Sysmon:
```
Comment (hitenkoku) [inline config/geoip_field_mapping.yaml:32]: @YamatoSecurity  Sysmon is channnel abbrevation. Doesn't it make the following SUGGESTION content when combined with the other descriptions?

```suggestion
  - Microsoft-Windows-Sysmon/Operational: 
```

**Review by hitenkoku — CHANGES_REQUESTED** [review]
@YamatoSecurity  Thank you for PR.

Would you please check the description of Sysmon in geoip_field_mapping?

File: config/geoip_field_mapping.yaml (hunk 1 above)
Comment (YamatoSecurity) [reply config/geoip_field_mapping.yaml]: Thank you! I forget it was an abbreviation.

**Review by hitenkoku — APPROVED** [review]
LGTM

**Pull request merged**